sdk/Lusid.Sdk.Tests/tutorials/Ibor/CutLabels.cs

```diff
@@ -0,0 +1,289 @@
+
+            // Retrieve holdings at LondonClose today (5pm local time)
+            // This will mean that the 4th transaction will not be included, demonstrating how cut labels work across time zones
+            getHoldingsCutLabel = Cut_Label_Formatter(_currentDate, code["LDNClose"]);
+            holdings = _transactionPortfoliosApi.GetHoldings(
+                scope: scope,
+                code: portfolioCode,
+                effectiveAt: getHoldingsCutLabel
+            );
+
+            // check that holdings are as expected after transactions for each instrument
+            holdings.Values.Sort((h1, h2) => String.Compare(h1.InstrumentUid, h2.InstrumentUid, StringComparison.Ordinal));
+            Assert.That(holdings.Values.Count, Is.EqualTo(4));
+            _testDataUtilities.AssertCashHoldings(
+                holdings: holdings,
+                index: 0,
+                instrumentId: currencyLuid,
+                units: (decimal)70000.0
+            );
+
+            _testDataUtilities.AssertHoldings(
+                holdings: holdings,
+                index: 1,
+                instrumentId: instrument1,
+                units: (decimal)200.0,
+                costAmount: (decimal)20100.0
+            );
+
+            _testDataUtilities.AssertHoldings(
+                holdings: holdings,
+                index: 2,
+                instrumentId: instrument2,
+                units: (decimal)200.0,
+                costAmount: (decimal)20200.0
+            );
+
+            _testDataUtilities.AssertHoldings(
+                holdings: holdings,
+                index: 3,
+                instrumentId: instrument3,
+                units: (decimal)200.0,
+                costAmount: (decimal)19900.0
+            );
+
+            _portfoliosApi.DeletePortfolio(scope, portfolioCode);
+        }
+
+        private string Cut_Label_Formatter(DateTime date, string cutLabelCode)
+        {
+            return date.ToString("yyyy-MM-dd") + "N" + cutLabelCode;
+        }
+
+        private void Create_Cut_Label(int hours, int minutes, string displayName, string description, string timeZone, Dictionary<string, string> codeDict)
+        {
+            // Create the time for the cut label
+            var time = new CutLocalTime(hours, minutes);
+
+            // Define the parameters of the cut label in a request
+            var request = new CreateCutLabelDefinitionRequest(
+                code: displayName + "-" + Guid.NewGuid().ToString().Substring(0, 4),
+                displayName: displayName,
+                description: description,
+                cutLocalTime: time,
+                timeZone: timeZone
+            );
+
+            // Add the codes of our cut labels to our dictionary
+            codeDict[request.DisplayName] = request.Code;
+
+            // Send the request to LUSID to create the cut label
+            var result = _apiFactory.Api<CutLabelDefinitionsApi>().CreateCutLabelDefinition(request);
```

Comment:
I think this is going to fail second time round as it isn't an "upsert" and we haven't deleted the previous ones

Reply:
It doesn't because of the GUIDs mentioned below. I'll add a check as you suggest there.